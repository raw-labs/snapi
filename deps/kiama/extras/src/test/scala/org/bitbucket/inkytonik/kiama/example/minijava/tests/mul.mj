class Mul {
    public static void main () {
        System.out.println (5 * 8);
    }
}
