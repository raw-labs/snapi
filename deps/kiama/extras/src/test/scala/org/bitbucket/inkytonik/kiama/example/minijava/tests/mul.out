.source src/test/scala/org/bitbucket/inkytonik/kiama/example/minijava/tests/mul.mj
.class public Mul
.super java/lang/Object

.method <init>()V
.limit stack 1
.limit locals 1
    aload_0
    invokespecial java/lang/Object/<init>()V
    return
.end method

.method public static main([Ljava/lang/String;)V
.limit stack 3
.limit locals 2
    getstatic java/lang/System/out Ljava/io/PrintStream;
    iconst_5
    bipush 8
    imul
    invokevirtual java/io/PrintStream/println(I)V
    return
.end method

